doc/nrf/known_issues.rst

```diff
@@ -483,60 +484,26 @@ KRKNWK-6408: ``diag`` command not supported
 Zigbee
 ======
 
-.. rst-class:: v1-9-1 v1-9-0
+.. rst-class:: v2-0-0
 
-KRKNWK-12937: Activation of Sleepy End Device must be done at the very first commissioning procedure for light switch sample
-   After programming the :ref:`Zigbee light switch <zigbee_light_switch_sample>` sample and its first commissioning, Zigbee End Device joins the Zigbee network as a normal End Device. Pressing **Button 3** does not switch the device to the Sleepy End Device configuration.
+KRKNWK-14024 Fatal error when the network coordinator factory resets in the Identify mode
+  A fatal error occurs when the :ref:`Zigbee network coordinator <zigbee_network_coordinator_sample>` triggers factory reset in the Identify mode.
 
-   **Workaround:** Keep **Button 3** pressed during the first commissioning procedure.
+  **Workaround:** Modify your application, so that the factory reset is requested only after the Identify mode ends.
 
-.. rst-class:: v1-9-1 v1-9-0
+.. rst-class:: v2-0-0 v1-9-1 v1-9-0
 
-KRKNWK-12522: Incorrect Read Attributes Response on reading multiple attributes when the first attribute is unsupported
-   When reading multiple attributes at once and the first one is not supported, the Read Attributes Response contains two records for the first supported attribute.
-   The first one record has the Status field filled with Unsupported Attribute whereas the second record contains actual data.
+KRKNWK-12937: Activation of Sleepy End Device must be done at the very first commissioning procedure for Zigbee light switch sample
+   After programming the :ref:`Zigbee light switch <zigbee_light_switch_sample>` sample and its first commissioning, Zigbee End Device joins the Zigbee network as a normal End Device. Pressing **Button 3** does not switch the device to the Sleepy End Device configuration.
```

Comment:
```suggestion
   After programming the :ref:`zigbee_light_switch_sample` sample and its first commissioning, Zigbee End Device joins the Zigbee network as a normal End Device. Pressing **Button 3** does not switch the device to the Sleepy End Device configuration.
```

Reply:
There is a colon in the raw link, and so I'm removing the colon on purpose by using custom name for the ref.